equipment-models/src/main/java/com/telecominfraproject/wlan/equipment/models/events/EquipmentChangedEvent.java

```diff
@@ -10,9 +13,20 @@
  */
 public class EquipmentChangedEvent extends EquipmentEventWithPayload<Equipment> implements HasLocationId {
     private static final long serialVersionUID = 7142209997917559985L;
+    private EquipmentChangeType equipmentChangeType;
```

Comment:
In original discussions, my understanding was EquipmentChangeType would be { ALL, AUTO_CHANNEL_ONLY, MANUAL_CHANNEL_ONLY, UNSUPPORTED} with the default being ALL. In this way the EquipmentChangedEvent wouldn't need extra fields for newPrimaryChannels and newBackupChannels. The provisioning-sp would know which fields to use in the event Equipment record based on EquipmentChangeType

Reply:
The extra fields means the event has a duplicate set of channels, one set in Equipment and another in newPrimaryChannels and newBackupChannels. I think we want to keep one source for the data, only the Equipment record.

In future, we could have other targeted commands for equipment updates. If we can avoid duplicating attributes then the event and processing will (hopefully) be cleaner.

Reply:
We don't have single autoChannelSelection control, it is radioType based

Reply:
Also CEGWNewChannelRequest needs newPrimaryChannels and newBackupChannels to pass the changes to AP.